int32 poseId
float64 poseX
float64 poseY
